examples/proxy/record-encryption/load-balancer/README.md

```diff
@@ -1,60 +1,70 @@
 # Streams for Apache Kafka Proxy Record Encryption, exposed by External Load Balancer
```

Comment:
Similar to what I said on #504 do we really need examples for both load balancer and cluster IP?

Reply:
Customers might have on cluster of off cluster apps so I wanted to help either group get going quickly.

I wish we had multiple listeners (and an operator).